### ISSUE TITLE
test set 질문

### ISSUE THREAD

**Issue by elzino** [Naver-AI-Hackathon/AI-Speech#33]
**Informations**
#### CLI
- **OS (Mac, Windows, Linux, etc) and version**: window10
- **client version(please show nsml --version)**:  
Built: 2019-09-06T11:25:12+09:00
Revision: 7b0bb1503fa9b3994273bd258bc68b31272907cb (not-modified)

#### WEB
- **Brower (chrome, firfox, etc)**: chrome

**What is your NSML login ID?**
elzino

**Question** 
공개된 training set과 test set이 같은 환경에서 녹음되었나요?
test set에 관한 정보는 대략이라도 공개 안하는게 원칙인가요?

**Comment by nsml-user** [comment]
> **Informations**
> 
> #### CLI
> * **OS (Mac, Windows, Linux, etc) and version**: window10
> * **client version(please show nsml --version)**:
>   Built: 2019-09-06T11:25:12+09:00
>   Revision: 7b0bb1503fa9b3994273bd258bc68b31272907cb (not-modified)
> 
> #### WEB
> * **Brower (chrome, firfox, etc)**: chrome
> 
> **What is your NSML login ID?**
> elzino
> 
> **Question**
> 공개된 training set과 test set이 같은 환경에서 녹음되었나요?
> test set에 관한 정보는 대략이라도 공개 안하는게 원칙인가요?

네. 같은 환경이라고 보시면 되겠습니다.
